src/main/java/htsjdk/variant/variantcontext/Genotype.java

```diff
@@ -556,6 +556,8 @@ public Object getAnyAttribute(final String key) {
             return Collections.EMPTY_LIST;
```

Comment:
While you're here, can you fix the javadoc for this method?  There's a verb missing.  "allows you to [what?] specific keys"